src/components/Form/Form.mdx

```diff
@@ -32,9 +32,13 @@ handle layout and spacing for child input fields. `FormRow` and `FormField`
 components should be used to build the `<form>` structure and to arrange input
 fields as needed.
 
+You are expected to pass `action` and `method` attributes to specify where form
+data should be sent and the method by which form will be submitted ("get" or
+"post").
```

Comment:
Specify that this is true for forms submitted through server-side actions. If a consuming app submits on the client side, the button's `onSubmit` takes precedence.